application/run-tests.sh

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
ah, I didn't realize that.  Ok, I'll add it back in when we get to CI then.  Thanks.